### PR TITLE
Added web pages generator module and info page to the WakeUp server

### PR DESCRIPTION
**Pull request by frsela**

### CODE REVIEW

**Comment by willyaranda** [comment]
I really like this. I hope a good framework will came for Node.js like Django por Python. Still road to ride.

File: src/ns_ua/ws_server.js [hunk 1]
```diff
@@ -235,16 +236,24 @@ server.prototype = {
...
+              var p = new pages();
+              p.setTemplate('views/aboutWS.tmpl');
+              text = p.render(function(t) {
+                switch (t) {
+                  case '{{GIT_VERSION}}':
+                    return require('fs').readFileSync('version.info');
+                  case '{{MODULE_NAME}}':
+                    return 'User Agent Frontend';
+                  case '{{PARAM_TOKENSGENERATED}}':
+                    return this.tokensGenerated;
```
Comment (willyaranda) [inline src/ns_ua/ws_server.js:248]: Is `this` working? I think you need to `.bind(this)` at the end of the anonymous function.

File: src/views/aboutWS.tmpl [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<html>
+  <head>
+    <title>Push Notification Server - {{MODULE_NAME}}</title>
+  </head>
+  <body>
+    <h1>Push Notification Server - {{MODULE_NAME}}</h1>
+    <p>&copy; Telef&oacute;nica Digital, 2012</p>
```
Comment (willyaranda) [inline src/views/aboutWS.tmpl]: BTW, we need to encode to HTML entities, can't we use just pure UTF-8 with proper symbols? :P ©

File: src/ns_ua/ws_server.js (hunk 1 above)
Comment (frsela) [inline src/ns_ua/ws_server.js:248]: You're right ... I uploaded a previous version and I didn't commited the bind ... :\ - fixed

File: src/views/aboutWS.tmpl (hunk 2 above)
Comment (frsela) [inline src/views/aboutWS.tmpl]: Agree. Fixed

**Comment by frsela** [comment]
> I really like this. I hope a good framework will came for Node.js like Django por Python. Still road to ride.

Me too, but if it doesn't exist we should create it ;) - What can go wrong? ;)

**Pull request merged**